doc/source/flexibility/binlog_space.rst

```diff
@@ -0,0 +1,48 @@
+.. _binlog_space:
+
+============================================================
+Limiting the disk space used by binary log files
+============================================================
+
+It is a challenge to control how much disk space is used by the binary logs. The size of a binary log can vary because a single transaction must be written to a single binary log and cannot be split between multiple binary log files.
```

Comment:
between multiple binary log files -> among multiple binary log files

Reply:
No. Between is the better word choice for this.